client/app/components/layout/SideNav.jsx

```diff
@@ -1,12 +1,14 @@
 import moment from 'moment';
-import React from 'react';
+import React, { PropTypes } from 'react';
 import { Button } from 'react-bootstrap';
+import { LinkContainer, IndexLinkContainer } from 'react-router-bootstrap';
+import { DATE_PARAM_FORMAT } from 'helpers/DateHelper';
 
 import RefreshButton from './RefreshButton';
 
 export default class SideNav extends React.Component {
   static propTypes = {
-    date:      React.PropTypes.string.isRequired,
+    date:      PropTypes.oneOfType([PropTypes.instanceOf(Date), PropTypes.string]).isRequired,
```

Comment:
Can we just inject one of these types, eg. Date? It's always confusing when you have to handle more than one type